exist-core/src/test/xquery/xsuite-declare-xqsuite.xqm

```diff
@@ -0,0 +1,35 @@
+(:
+ : eXist-db Open Source Native XML Database
+ : Copyright (C) 2001 The eXist-db Authors
+ :
+ : info@exist-db.org
+ : http://www.exist-db.org
+ :
+ : This library is free software; you can redistribute it and/or
+ : modify it under the terms of the GNU Lesser General Public
+ : License as published by the Free Software Foundation; either
+ : version 2.1 of the License, or (at your option) any later version.
+ :
+ : This library is distributed in the hope that it will be useful,
+ : but WITHOUT ANY WARRANTY; without even the implied warranty of
+ : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ : Lesser General Public License for more details.
+ :
+ : You should have received a copy of the GNU Lesser General Public
+ : License along with this library; if not, write to the Free Software
+ : Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ :)
+xquery version "3.1";
+
+(:~
+ : Test for using XQSuite from XSuite via `declare namespace`.
```

Comment:
XSuite = XQSuite?
If this is just a typo please correct it as it has the potential to confuse people (it did confuse me).

Reply:
See https://github.com/eXist-db/exist/blob/develop/exist-core/src/main/java/org/exist/test/runner/XSuite.java#L47-L59.

Reply:
@line-o XSuite is the Java test runner that calls XQSuite. This makes no change to eXide.